source/Halibut/Transport/DiscoveryClient.cs

```diff
@@ -42,7 +42,11 @@ public async Task<ServiceEndPoint> DiscoverAsync(ServiceEndPoint serviceEndpoint
 #if NETFRAMEWORK
                             // TODO: ASYNC ME UP!
                             // AuthenticateAsClientAsync in .NET 4.8 does not support cancellation tokens. So `cancellationToken` is not respected here.
-                            await ssl.AuthenticateAsClientAsync(serviceEndpoint.BaseUri.Host, new X509Certificate2Collection(), SslProtocols.Tls | SslProtocols.Tls11 | SslProtocols.Tls12, false);
+                            await ssl.AuthenticateAsClientAsync(
+                                serviceEndpoint.BaseUri.Host,
+                                new X509Certificate2Collection(),
+                                SslProtocols.Tls | SslProtocols.Tls11 | SslProtocols.Tls12 | SslProtocols.Tls13,
```

Comment:
Nit: It might make sense to pull this list out to a single place and perhaps make it user configurable within the `HalibutRuntimeBuilder`